lib/generators/templates/setup/brewfile

```diff
@@ -0,0 +1,9 @@
+# https://formulae.brew.sh/formula/vips
+brew "vips"
+
+# https://formulae.brew.sh/formula/redis
+brew "redis"
+
+# https://formulae.brew.sh/formula/postgresql@15
+# Change version to match production
+brew "postgresql@15"
```

Comment:
Should we raise early if we're not using Postgres? I ask because we've decided as an organization we prefer PostgreSQL.

https://github.com/thoughtbot/dotfiles/pull/728